README.md

```diff
@@ -36,7 +36,13 @@ An [Elastic Email][] adapter for the [Bamboo][] email app for Elixir.
       api_key: "my-api-key"
     ```
 
-4.  Follow the Bamboo [Getting Started Guide][getting_started].
+4. Follow the Bamboo [Getting Started Guide][getting_started].
+
+5. To use ElasticEmails [postBack] functionality you can place a value in the `Email#private` parameter:
```

Comment:
Two small things on this line. This should be `To use ElasticEmail's [postBack][] functionality…`. I’m curious why you implemented this as `:elastic_custom_vars` with a map argument (only to implement a handler for `:post_back`) as opposed to `:elastic_email_post_back` with a `String.t()` argument. I don’t disagree with the pattern (it will work better for the future on this), just curious as we should probably support more than just `post_back`.

Reply:
I implemented it this way based on Bamboo's MailGun adapter.  My line of thinking was similar to what you mentioned and that's to support future expansion of other custom ElasticEmail parameters.